docs/content/latest/reference/configuration/yugabyted/errors-glossary.md

```diff
@@ -0,0 +1,65 @@
+---
+title: yugabyted Errors and Warnings
+linkTitle: Errors and Warnings
+description: yugabyted CLI errors and warnings
+section: REFERENCE
+menu:
+  latest:
+    identifier: yugabyted-errors-warnings
+    parent: configuration
+    weight: 2455
+isTocNested: true
+showAsideToc: true
+---
+
+This page explains errors and warnings that you may experience while running a YugabyteDB cluster using yugabyted CLI and ways to resolve them.
+
+| CLI Message | CLI Command (Task) | Error/Warning |
+| --- | --- | --- |
+| [open files ulimits value set low](#open-files-ulimits-value-set-low) | installation | Warning |
+| [max user processes value set low](#max-user-processes-value-set-low) | installation | Warning |
+| [Transparent hugepages disabled](#transparent-hugepages-disabled) | installation | Warning |
+| [Six loopback IPs are required](#six-loopback-ips-are-required) | installation | Warning |
+| [ntp package missing](#ntp-package-missing) | installation | Warning |
+
+## open files ulimits value set low
```

Comment:
Sentence case for titles unless there's a really good reason not to...easier to read.
```suggestion
## Open files ulimits value set low
```